app/lib/feature_flags.rb

```diff
@@ -1,6 +1,7 @@
 class FeatureFlags
   ENABLED_AFTER_DATE = {
     example_2125_flag: "2125-01-01",
+    el_809: "2023-04-10",
```

Comment:
a more descriptive name would be better, such as dependance_allowance_increase, so we know what it is without needing to reference the jira ticket